src/NzbDrone.Core/Indexers/Definitions/Cardigann/CardigannBase.cs

```diff
@@ -613,14 +614,22 @@ protected string ApplyFilters(string data, List<FilterBlock> filters, Dictionary
                     case "timeparse":
                     case "dateparse":
                         var layout = (string)filter.Args;
-                        try
+
+                        if (layout.Contains("yy") && DateTime.TryParseExact(data, layout, CultureInfo.InvariantCulture, DateTimeStyles.None, out var parsedDate))
```

Comment:
Can we just remove the replaces in parseGoLang and enforce the correct format in definitions with yml?